content/en/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/unity.md

```diff
@@ -23,28 +23,25 @@ further_reading:
 
 If you experience unexpected behavior with Datadog RUM, use this guide to resolve issues quickly. If you continue to have trouble, contact [Datadog Support][1] for further assistance.
 
-## Set sdkVerbosity
+## Set sdkVerbosity for easier debugging
 
-If you're able to run your app, but you are not seeing the data you expect on the Datadog site, try adding the following to your code as part of initialization :
+If you're able to run your app, but you are not seeing the data you expect on the Datadog site, try adding the following to your code as part of initialization:
 
-```cs
+{{< code-block lang="cs" >}}
 DatadogSdk.Instance.SetSdkVerbosity(CoreLoggerLevel.Debug);
-```
+{{< /code-block >}}
 
 This causes the SDK to output additional information about what it's doing and what errors it's encountering, which may help you and Datadog Support narrow down your issue.
 
 ## SDK not sending data
 
-If you're not seeing any data in Datadog, first make sure you are running your app on an iOS or Android simulator, emulator, or device, and not from the editor.
+If you're not seeing any data in Datadog:
 
-<div class="alert alert-info">
-Datadog does not currently support sending data fro the Unity Editor, only from iOS and Android simulators, emulators, and devices.
-</div>
-
-Next, check that you have set the `TrackingConsent` as part of your initialization. Tracking consent is set to `TrackingConsent.Pending` during initialization,
+1. Make sure you are running your app on an iOS or Android simulator, emulator, or device, and not from the editor. Datadog does not support sending data from the Unity Editor, only from iOS and Android simulators, emulators, and devices.
```

Comment:
Is there any way we can call more attention to the lack of Editor support. This is a common issue we've had and I don't want it missed.

Reply:
Sure, I'll put it in an alert